src/Illuminate/Support/Facades/Schema.php

```diff
@@ -21,6 +21,7 @@
  * @method static void create(string $table, \Closure $callback)
  * @method static void drop(string $table)
  * @method static void dropIfExists(string $table)
+ * @method static void dropIndexIfExists(string $table,string $index)
```

Comment:
Transforming the `dropIndexIfExists` to accept array, this doc should be updated, or I'm wrong?

Reply:
@devajmeireles doc updated.
thanks for the review